disputer/test/index.js

```diff
@@ -0,0 +1,89 @@
+const { toWei, utf8ToHex } = web3.utils;
+
+// Script to test
+const Poll = require("../index.js");
+const { Disputer } = require("../disputer.js");
+
+// Helper client script
+const { ExpiringMultiPartyClient } = require("../../financial-templates-lib/ExpiringMultiPartyClient");
+const { GasEstimator } = require("../../financial-templates-lib/GasEstimator");
+
+// Contracts and helpers
+const ExpiringMultiParty = artifacts.require("ExpiringMultiParty");
+const Finder = artifacts.require("Finder");
+const IdentifierWhitelist = artifacts.require("IdentifierWhitelist");
+const MockOracle = artifacts.require("MockOracle");
+const TokenFactory = artifacts.require("TokenFactory");
+const Token = artifacts.require("ExpandedERC20");
+
+contract("Disputer.js", function(accounts) {
+  const disputeBot = accounts[0];
+  const contractCreator = accounts[1];
+
+  let collateralToken;
+  let emp;
+  let mockOracle;
+
+  before(async function() {
+    collateralToken = await Token.new({ from: contractCreator });
+    await collateralToken.addMember(1, contractCreator, {
+      from: contractCreator
+    });
+
+    // Create a mockOracle and finder. Register the mockMoracle with the finder.
+    mockOracle = await MockOracle.new(IdentifierWhitelist.address, {
+      from: contractCreator
+    });
```

Comment:
Technically I think we can do without much of this setup as well, but I understand if you think we'll be adding more tests here that depend on it.

Reply:
True, I am removing mock oracle logic and `addMember`, will do so from liquidator bot test as well

Reply:
done